src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import "./variables";
+@import "./mixins";
+
+@import "./card";
+@import "./stars";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
+  @include zero-margins-paddings;
+  font-family: $cardsFontStyle;
+  font-weight: 400;
 }
```

Comment:
Consider moving general styling to a separate file to keep your main file clean

Reply:
Thank you for your help in solving the task!